zt_cli/cli.py

```diff
@@ -205,7 +210,7 @@ def notebook(
     """
     Start the Zero-True application.
     """
-
+    set_zt_path
```

Comment:
Parentheses missing

Reply:
sorry, for not checking properly will update this